.changeset/config.json

```diff
@@ -12,7 +12,7 @@
   "access": "public",
   "baseBranch": "origin/main",
   "updateInternalDependencies": "patch",
-  "ignore": ["gatsby-plugin-clerk"],
+  "ignore": [],
```

Comment:
Temporarily removing the gatsby SDK to ignored list so we can publish the notice to NPM